components/canvas/Flow.tsx

```diff
@@ -145,6 +153,7 @@ const Flow = ({ apiNodes, apiEdges, userCanEdit }: CanvasProps) => {
         nodeDragThreshold={15}
         onNodeContextMenu={onNodeContextMenu}
         onPaneContextMenu={onPaneContextMenu}
+        preventScrolling={!selectedNode}
```

Comment:
After testing this a bit more I think we should change this to {selectedNode && hoveredNode?.id !== selectedNode?.id}. Then the user can still zoom in/out outside of the selected node.

Reply:
fixed!